### ISSUE TITLE
Octave control only goes between 0 and 1 when used with BitWig Studio

### ISSUE THREAD

**Issue by eoghanmurray** [artfwo/andes#22]
I noticed from the paper that the octave parameter can go beyond `1` 
however when loaded as a plugin with BitWig Studio (`Andes1-v0.2-linux64`), the knob is limited to max `1.0`.
Not sure if that's the case with other DAW hosts on Linux!

**Comment by artfwo** [comment]
Yes, octaves can go up to 16. It's working fine with a few other hosts, but I will check with BitWig, thanks for reporting!

**Comment by artfwo** [comment]
@eoghanmurray seems to work for me (screenshot below), what version of BitWig are you using?

![image](https://user-images.githubusercontent.com/689741/99577467-ad7a9280-29db-11eb-8235-6b586e4f6c92.png)

**Comment by eoghanmurray** [comment]
I'm using BitWig 3.2

I was not seeing those nice bespoke controls from your screenshot though!  
Checking again, I pressed the 'Show Plug-In Window' icon, but it's impossible to make that window any bigger.

So my original observation was that the native bitwig control for `octaves` doesn't go further than 1 ... this appears to be visible in your screenshot too (purple box in the screenshot below):

![image](https://user-images.githubusercontent.com/156780/99678107-5f649e00-2a72-11eb-9191-8fab8cddf2eb.png)

**Comment by artfwo** [comment]
I've been checking this with BitWig 3.2.8, but can't reproduce this. Do you always see the tiny window when clicking "show plugin window" button?

**Comment by eoghanmurray** [comment]
Yes, it still happens after restart!
The only thing I can think of is that I've got a high DPI screen.
Oh well, it could be something to do with BitWig :(
